test/solc/CommandLineInterface.cpp

```diff
@@ -1018,6 +1025,72 @@ BOOST_AUTO_TEST_CASE(cli_include_paths)
 	BOOST_TEST(result.reader.basePath() == expectedWorkDir / "base/");
 }
 
+BOOST_AUTO_TEST_CASE(enum_only)
+{
+	TemporaryDirectory tempDir({"base/"}, TEST_CASE_NAME);
+	TemporaryWorkingDirectory tempWorkDir(tempDir);
+
+	string const mainContractSource =
+		"// SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\n\nenum Status {\n		test\n}\n";
```

Comment:
```suggestion
	string const mainContractSource = R"(
			// SPDX-License-Identifier: GPL-3.0
			pragma solidity >=0.0;
	
			enum Status { test }
		)";
```

Same in the other test.